Documentation/project-docs/dogfooding.md

```diff
@@ -117,6 +144,18 @@ $ dotnet publish
 $ bin\Debug\netcoreapp2.0\win7-x64\publish\App.exe
 ```
 
+### Using an isolated SDK
+
+It is possible to install the SDK in a completely isolated fashion, if you do not want to make global changes.
+
+1. Download the zip or tarball from https://github.com/dotnet/cli  (for example, https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/dotnet-sdk-latest-win-x64.zip ) into a new folder.
```

Comment:
shouldn't this point to https://github.com/dotnet/core-setup instead of cli?

Reply:
core-setup has downloads of the runtime (core-setup, coreclr, and corefx). cli has the whole SDK in it. you can use the core-setup package if you have a reasonably compatible SDK installed globally and don't mind it using that. I figured I would document using the CLI one since it's more likely to be successful even though it's a little bigger.